src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,6 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Don’t think it was necessary to reformat everything in this file for this change.